app/code/Magento/Multishipping/etc/config.xml

```diff
@@ -13,5 +13,10 @@
                 <checkout_multiple_maximum_qty>100</checkout_multiple_maximum_qty>
             </options>
         </multishipping>
+        <sales>
+            <minimum_order>
+                <multi_address_error_message>All shippable products have been removed from your order</multi_address_error_message>
```

Comment:
I think this message should be more general and just highlight an error, at the moment it sounds like the system has removed the items from the cart. I would recommend something like: "The current cart does not match multi shipping criteria, please review or contact the store administrator".

On the issue-specific flow, the check is being done is on `app/code/Magento/Multishipping/Helper/Data - isMultishippingCheckoutAvailable()` and that checks several different properties that are relevant for multi shipping.

Reply:
@gabrieldagama Adjusted and PR is ready for review.